scripts/vlm/neva_generate.py

```diff
@@ -190,7 +200,12 @@ def main(args) -> None:
     if args.load_from_hf:
         model = fabric.import_model("hf://llava-hf/llava-1.5-7b-hf", LlavaModel)
     else:
-        model = LlavaModel(Llava15Config7B(), tokenizer=hf_tokenizer)
+        config = Llava15Config7B()
+        new_transformer_layer_spec = get_gpt_modelopt_spec(
+            config.language_transformer_config, local_core_attention=False, remap_te_layernorm=True
+        )
+        config.language_transformer_config.transformer_layer_spec = new_transformer_layer_spec
```

Comment:
this should be done only `if args.enable_quantization`

Reply:
This is a great suggestions, let me add this one.